draft-wang-ppm-differential-privacy.md

```diff
@@ -602,17 +631,19 @@ measurement `D` from the measurement from an average Client.
 
 ### Reference Implementation
 
-A reference implementation of binary randomized response can be found below.
+A reference implementation of symmetric RAPPOR can be found below.
 
 > TODO: We need to align with `Xof` specified in the VDAF draft. Currently
-> we get away with just using `random.random()` in python to make the randomized
-> response sampler functional.
+> we get away with just using `random.random()` in python to make the symmetric
+> RAPPOR functional.
 > TODO: We could make the sampler more efficient if we use binomial.
 
+>KT: Should we say here what
```

Comment:
Not sure what @kunal-talwar wants to say here. Waiting to hear back from him.

Reply:
From Kunal: describe what each of the three methods of `DpMechanism` does. I've clarified them now.